netshare/generate/model_generate.py

```diff
@@ -1,89 +1,117 @@
 import copy
-import subprocess
-import sys
-import time
 import os
-import json
-import importlib
-import random
-import pandas as pd
-import socket
-import struct
-import ipaddress
-import argparse
+import time
+from pathlib import Path
+from typing import Dict, List
 
 import numpy as np
 import pandas as pd
-
-import netshare.ray as ray
-from pathlib import Path
 from tqdm import tqdm
-from scapy.all import IP, ICMP, TCP, UDP
-from scapy.all import wrpcap
-from scipy.stats import rankdata
-from pathlib import Path
+
+import netshare.utils.ray as ray
+from netshare import models
+from netshare.utils.logger import TqdmToLogger, logger
+from netshare.utils.model_configuration import create_chunks_configurations
+from netshare.utils.paths import get_generated_data_log_folder
+
+
+def model_generate() -> None:
+    configs, config_group_list = create_chunks_configurations(generation_flag=True)
+
+    logger.info("Start generating attributes")
+    ray.get(
+        [
+            generate_data.remote(
+                config=config,
+                given_data_attribute_flag=False,
+            )
+            for config in configs
+        ]
+    )
+    time.sleep(10)
```

Comment:
Why do we need the sleep?

Reply:
I don't know, it was in the original code!
I thought that it might have an internal logic that I'm missing 😂 
As long as we don't have good tests, I don't feel comfortable with deleting parts that I don't understand. Now that you highlighted it, I'll try to remove it and validate manually that everything is ok

Reply:
Previously, sleep is used to help ray to sync the jobs. I understand that ray has mechanism which can synchronize distributed jobs as needed. But from my personal experiences, there's possibility (not large) that ray would work not as expected. So i just add `sleep` even though it's not elegent. We can also check if we can run correctly without `sleep` when using the latest version of ray.